### DISCUSSION TITLE
CockroachDB serverless calls are too slow

### DISCUSSION THREAD

**Discussion by malekjaroslav** [vercel/community#1328] [Help]
### Summary

Hi all,

I use CockroachDB serverless in Frankfurt region and have a Vercel function region also set to Frankfurt.  My app  was previously hosted on a server instance in the same region and the response time was instant. Calls that took just a few milliseconds now take almost a second. 

Do you have some ideas on improving this? The app's UX has degraded significantly after moving to Vercel.

Thank you!

**Comment by leerob** [comment]
Hey there, can you share some more information?

1. Links to your deployed application
2. Screenshots of the Cockroach setup so we can understand and confirm the region
3. Any other information that would be helpful for us to look at

Thank you!

https://vercel.com/guides/using-databases-with-vercel

**Reply by malekjaroslav** [reply]
Hi @leerob, thanks for the instant reply!

I use Remix with Prisma and Cockroach DB, my app is here - https://vercel.com/renton-app/renton (app.renton.cz)
Cluster settings: <img width="432" alt="Screenshot 2023-01-22 at 22 17 51" src="https://user-images.githubusercontent.com/7753942/213940960-ab4d6da5-f4a4-4722-96ad-4851b2eff67c.png">

**Reply by leerob** [reply]
Thank you!

> Calls that took just a few milliseconds now take almost a second.

Can you share some examples and how you're measuring this?
